pkg/manager/config/config.go

```diff
@@ -449,6 +450,11 @@ func GetCustomServedFolders() URLMap {
 }
 
 // Interface options
+func GetMenuItems() []string {
+	viper.SetDefault(MenuItems, []string{"scenes", "images", "movies", "markers", "performers", "studios", "tags"})
```

Comment:
Instead of setting a default, I think this should just get the string slice, and return the default if the slice is empty.

Reply:
Do I understand correctly that that would make it impossible to set the option to the empty slice explicitly? That could be a bit surprising behaviour when the user intentionally disables all menu items in settings (not sure why they would want that, but hey). Please let me know what you think, I'm happy to change it.

Reply:
Good point. Instead, call `viper.IsSet` to determine if the key is set. If it isn't, return the default list, otherwise return the string slice for the key.

Reply:
Sure thing, done.